pkg/interfaces/contracts/vault/IVaultErrors.sol

```diff
@@ -48,6 +48,12 @@ interface IVaultErrors {
      */
     error TokenAlreadyRegistered(IERC20 token);
 
+    /**
+     * @dev Token were already registered (i.e., it is a duplicate in the pool).
+     * @param tokens The tokens from the storage
+     */
+    error TokensAlreadyRegistered(IERC20[] tokens);
```

Comment:
I don't think we need another error here (see below), and it's confusing to have two very similar errors (Token vs. Tokens)